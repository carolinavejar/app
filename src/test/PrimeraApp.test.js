import React from 'react';
import PrimeraApp  from '../PrimeraApp';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

describe("Pruebas de Primera App .... .test.js", ()=> {
    test('debe retornar Hola soy Carito', () => {
        const saludo = `Hola soy Carito`;
        const { getByText } = render ( < PrimeraApp saludo = { saludo } />);

        expect ( getByText(saludo) ).toBeInTheDocument();
    });

})