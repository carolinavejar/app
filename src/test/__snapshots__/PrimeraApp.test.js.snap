// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de Primera App .... .test.js debe de mostrar <PrimeraApp /> correctamente 1`] = `
<Fragment>
  <h1>
       
    Hola soy Carito
     !!!  
  </h1>
  <p>
      
    sub por defecto
     
  </p>
</Fragment>
`;

exports[`Pruebas de Primera App .... .test.js debe de mostrar subtitulo de props 1`] = `
<Fragment>
  <h1>
       
    Hola soy Carito
     !!!  
  </h1>
  <p>
      
    sub de prueba
     
  </p>
</Fragment>
`;
